tests/migrations/test-migration-Mig004.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from mock import patch
+
+from octoprint_mrbeam.migration import Mig004DisableDebugLogging
+import unittest
+
+
+class TestMigrationMig004(unittest.TestCase):
```

Comment:
No need to change now, but I thought we agreed to use pytest :thinking:

Reply:
I copied the existing test